ee_tests/local_run_EE_tests.sh

```diff
@@ -101,8 +58,11 @@ main() {
 
   local log_file="${SCRIPT_DIR}/functional_tests.log"
 
-  start_webdriver "$log_file"
-  wait_for_webdriver
+  # NOTE: DO NOT start and kill webdriver if it is already running
```

Comment:
@ldimaggi Please note the change in behaviour. With this change, if you started a webdriver manually before running the script, it wouldn't kill the running server where if the webdriver isn't running, then it will start one and kill it at the end of the test run.